gatsby/content/projects/bridges/heisenbridge.mdx

```diff
@@ -0,0 +1,19 @@
+---
+layout: project
+title: Heisenbridge
+categories:
+ - bridge
+description: a bouncer style Matrix IRC bridge
+author: hifi
+home: https://github.com/hifi/heisenbridge
+language: Python
+license: MIT
+repo: https://github.com/hifi/heisenbridge
+room: "#heisenbridge:vi.fi"
+thumbnail: /images/irc.png
+maturity: Alpha
+featured: true
+bridges: IRC
+---
+
+A bouncer style Matrix IRC bridge with focus on simplicity for personal use on your own homeserver. No complex configuration files or databases required.
```

Comment:
[Compound adjectives](https://www.thefreedictionary.com/Compound-Adjectives.htm) use one or more hyphens as delimiters.

This is just a comment, not a review. I'm not a repo owner.

```suggestion
description: a bouncer-style Matrix IRC bridge
author: hifi
home: https://github.com/hifi/heisenbridge
language: Python
license: MIT
repo: https://github.com/hifi/heisenbridge
room: "#heisenbridge:vi.fi"
thumbnail: /images/irc.png
maturity: Alpha
featured: true
bridges: IRC
---

A bouncer-style Matrix IRC bridge with focus on simplicity for personal use on your own homeserver. No complex configuration files or databases required.
```

Reply:
Thank you for the heads up. I'm not a native speaker and this things slip through. I rebased and updated with your suggestion.